lib/zold/commands/alias.rb

```diff
@@ -27,10 +27,34 @@ def run(args = [])
       mine = Args.new(opts, @log).take || return
       command = mine[0]
       raise "A command is required, try 'zold alias --help'" unless command
+      case command
+      when 'set'    then set(mine[1], mine[2])
+      when 'remove' then remove(mine[1])
+      when 'show'   then show(mine[1])
+      end
+    end
+
+    # @todo #323:30min Implement the set command.
+    #  The set command expects 'wallet' which is the wallet id
+    #  and 'alias' (here called 'name' as alias is a Ruby keyword).
+    #  It writes this info into the alias file as described in #279.
+    def set(_wallet, _name)
```

Comment:
@botanicus do tests exist for these?

Reply:
Yes https://github.com/zold-io/zold/pull/330